### PR TITLE
added contributing.md documentation

### PR DESCRIPTION
**Pull request by ThomasAlxDmy**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+### How to contribute ?
+We will accept contributions from anybody assuming they respect our policies and help improve the platform. here's the different type of contribution we support.
+
+#### :lady_beetle: Issues
+[Issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues) are used to track tasks that contributors can help with. If an issue has a triage label, we haven't reviewed it yet, and you shouldn't begin work on it.
+
+If you've found something in the code that should be updated, search open issues to see if someone else has reported the same thing. If it's something new, open an issue. We'll use the issue to have a conversation about the problem you are having or want to fix.
+
+#### :hammer_and_wrench: Pull requests
+A [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) is a way to suggest changes in our repository. When we merge those changes, it will kick of a new build and push the new version to docker. :earth_africa:
```
[the remark this answers is not in the document]
Comment (ThomasAlxDmy) [reply CONTRIBUTING.md:28]: Seems like this should be automated.

[the remark this answers is not in the document]
Comment (agbpatro) [reply CONTRIBUTING.md]: There were few reasons:-
- We want to control if the new release should be a major, minor or patch
- Ideally, we didn't want to release for no-op PRs (updating readme, adding/updating tests without app logic changes)

[the remark this answers is not in the document]
Comment (ThomasAlxDmy) [reply CONTRIBUTING.md]: can we have a release file part of the repo and derive it from that?

[the remark this answers is not in the document]
Comment (aaronpkahn) [reply CONTRIBUTING.md]: I agree with @agbpatro.  The process for publishing a release is a single button click, and will reduce confusion to publish fewer of them (not on every commit)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Once you submit your PR, a Docs team member will review your proposal. We may ask questions or request additional information.
+- We may ask for changes to be made before a PR can be merged, either using [suggested changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) or pull request comments. You can apply suggested changes directly through the UI. You can make any other changes in your fork, then commit them to your branch.
+- As you update your PR and apply changes, mark each conversation as [resolved](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations).
+- If you run into any merge issues, checkout this [git tutorial](https://github.com/skills/resolve-merge-conflicts) to help you resolve merge conflicts and other issues.
+
+### Your PR is merged!
+
+Once your PR is merged, your contributions will be publicly visible on the [Fleet Telemetry github](https://github.com/teslamotors/fleet-telemetry).
```
Comment (agbpatro) [inline CONTRIBUTING.md:93]: Fix casing for `Github` here to make it consistent with other part of the file?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- [Finding ways to contribute to open source on GitHub](https://docs.github.com/en/get-started/exploring-projects-on-github/finding-ways-to-contribute-to-open-source-on-github)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+- [Collaborating with pull requests](https://docs.github.com/en/github/collaborating-with-pull-requests)
+
+
+## Getting started
+
+### How to contribute ?
+We will accept contributions from anybody assuming they respect our policies and help improve the platform. here's the different type of contribution we support.
```
Comment (agbpatro) [inline CONTRIBUTING.md:20]: nit: `different types of contribution we support.`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+#### :hammer_and_wrench: Pull requests
+A [pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) is a way to suggest changes in our repository. When we merge those changes, it will kick of a new build. Currently we will manually push the update to docker to ensure a good understanding of release should be a major, minor or patch. We might automate this process in the future.
+
+
+#### :question: Support
+We are a small team working hard to keep up with the documentation demands of a continuously changing product. Unfortunately, we can't help with support questions in this repository. If you are experiencing a problem unrelated to the code hosted here please [contact Tesla Support directly](https://https://www.tesla.com/support). Any issues, discussions, or pull requests opened here requesting support will be given information about how to contact Tesla Support, then closed and locked.
+
+#### :earth_asia: Translations
+
+This website is internationalized and available in multiple languages. The source content in this repository is written in English.
```
Comment (agbpatro) [inline CONTRIBUTING.md:36]: is this accurate? If yes, can we post link for translated content in the documentation for quick access?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
+# Welcome to Fleet Telemetry contributing guide <!-- omit in toc -->
+
+Thank you for investing your time in contributing to our project!
```
Comment (aaronpkahn) [inline CONTRIBUTING.md:3]: `Thank you for investing your time contributing to the project!` (remove "in" and "our")

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with the docs, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments).
+
+Issues created here should only target the Fleet Telemetry project, if you have any issues with your vehicle please [contact Tesla Support directly](https://https://www.tesla.com/support).
+
+DO NOT: When creating or responding to an Github Issue DO NOT post any personal information, account information, vehicle information or anything that could identify you. Stay focus on what the technical problems are with the backend.
```
Comment (aaronpkahn) [inline CONTRIBUTING.md:48]: "an" should be "a": `a Github Issue`.  Remove: `Stay focus on what the technical problems are with the backend.`

File: CONTRIBUTING.md (hunk 2 above)
Comment (aaronpkahn) [inline CONTRIBUTING.md:93]: may want to add `, and will be published in a future [release](https://github.com/teslamotors/fleet-telemetry/releases).`

File: CONTRIBUTING.md (hunk 4 above)
Comment (ThomasAlxDmy) [reply CONTRIBUTING.md:36]: This website is internationalized and available => github. The source content in this repository is written in English => fleet streaming

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+This website is internationalized and available in multiple languages. The source content in this repository is written in English.
+
+**We do not currently accept contributions for translated content**.
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with the docs, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments).
```
Comment (agbpatro) [inline CONTRIBUTING.md:44]: nit: `If you spot a problem within the repo`

**Review by agbpatro — APPROVED** [review]

**Pull request merged**